### ISSUE TITLE
Re-Assign VLJ Support Staff Case 

### ISSUE THREAD

**Issue by marvokdolor-gov** [department-of-veterans-affairs/caseflow#8582]
When testing access to the queue page for VLJ Support Staff I added myself to that team. I forgot to remove myself overnight and got assigned a case. 

I have removed myself from the team at Lowell's direction.

### Next step:
- reassign that now orphan case to another VLJ Support Staff team member.

## Acceptance Criteria
- Case previously assigned to Marvourneen Dolor 01/08/2018, now assigned to another member of the VLJ Support Staff team.

**Comment by marvokdolor-gov** [comment]
@lowellrex I'm wondering if you could/should also reassign Eugene Scott's cases since he's no longer at the Board and not showing up on the VLJ Support Staff team page. 
He is, however, showing up in the VLJ Support Staff [looker report](https://caseflow-looker.va.gov/looks/78) as being assigned 20 cases.

**Comment by lowellrex** [comment]
Assigned Marvo's task to a member of the VLJ support staff:
```ruby
rails c> u = User.find_by(email: "marvourneen.dolor@va.gov")
rails c> tasks = Task.where(assigned_to: u)
rails c> tasks.count
# 1
rails c> task = tasks.first
rails c> task.parent.children.count
# 1
# Create a new colocated task assigned to the next assignee
rails c> task.dup.tap { |t| t.assigned_to = ColocatedTaskDistributor.new.next_assignee; t.save! }
# Have we successfully created the new task?
rails c> task.parent.children.count
# 2
# Close the old task.
rails c> task.update!(status: Constants.TASK_STATUSES.completed)
# Did it work?
rails c> task.status
# "completed"
```

**Comment by lowellrex** [comment]
Will come back and reassign that Eugene Scott's last task after #8593 is live.

**Comment by lowellrex** [comment]
#8593 has been merged and I was able to reassign Eugene Scott's last task.
